tests/sentry/api/endpoints/test_organization_details.py

```diff
@@ -137,7 +137,7 @@ def test_with_projects(self):
             options.delete("store.symbolicate-event-lpq-never")
 
         # TODO(dcramer): We need to pare this down. Lots of duplicate queries for membership data.
-        expected_queries = 44
+        expected_queries = 45
```

Comment:
I'm confused by this test. Originally it failed saying it expected 44 queries but ran 45, so I changed it to 45 and now it says it expected 44 😑

Reply:
The test is run twice, once for the silo mode and once for the non silo mode -- I'm guessing the numbers vary based on the silo modality.  I can help with this.